classifurlr/har_utils.py

```diff
@@ -25,3 +25,10 @@ def har_entry_response_content(entry):
         return str(BeautifulSoup(text, 'lxml'))
     except Exception as e:
         raise NotEnoughDataError('Could not parse entry content')
+
+def get_total_size(entries):
+  size = 0
+  for entry in entries:
+    if entry['response']['content']['size'] > 0:
```

Comment:
Are all of these keys guaranteed to exist or do we risk throwing a `KeyError`?

Are we absolutely sure that the entries will be of type int or might we need to coerce them?

Reply:
http://www.softwareishard.com/blog/har-12-spec/#response

No risk I would say.

Reply:
Oh, well, really low risk, never say never :smile:

Reply:
OK. Please drop that link in as a comment so that future-us can be reassured (and can find where the key names came from). Then :rainbow:.